test/arbeidsuforhet/VurderingHistorikkTest.tsx

```diff
@@ -23,14 +23,29 @@ import {
 import { daysFromToday } from "../testUtils";
 import { tilDatoMedManedNavn } from "@/utils/datoUtils";
 import userEvent from "@testing-library/user-event";
+import { pengestoppStatusQueryKeys } from "@/data/pengestopp/pengestoppQueryHooks";
+import {
+  Sykepengestopp,
+  ValidSykepengestoppArsakType,
+} from "@/data/pengestopp/types/FlaggPerson";
+import { defaultSykepengestopp } from "@/mocks/ispengestopp/pengestoppStatusMock";
 
 let queryClient: QueryClient;
 
-const renderVurderingHistorikk = (vurderinger: VurderingResponseDTO[]) => {
+const renderVurderingHistorikk = (
+  vurderinger: VurderingResponseDTO[],
+  statuser: Sykepengestopp[] = []
```

Comment:
Få med variabelnavn også? 😊
```suggestion
  sykepengestopp: Sykepengestopp[] = []
```

Reply:
Klarte å snike seg forbi noe. Tenker jeg kaller den `sykepengestoppList` så er det konsekvent mtp. andre endringene som er gjort ellers.